### PR TITLE
Views for product page when any org is selected

### PR DESCRIPTION
**Pull request by ColeHiggins2**
Currently working on a view page for Content -> Product when any organization is select

Screenshot from 2021-12-01 15-09-09
ed

### CODE REVIEW

File: airgun/entities/organization.py [hunk 1]
```diff
@@ -55,6 +56,15 @@ def select(self, org_name):
         """Select necessary organization from context menu on the top of the page"""
         self.navigate_to(self, 'Context', org_name=org_name)
 
+    def select_org_page(self, org_name):
+        """Select necessary organization from menu on the the page.
```
Comment (swadeley) [inline airgun/entities/organization.py:60]:
```suggestion
        """Select necessary organization from menu on the page.
```

**Comment by omkarkhatavkar** [comment]
@ColeHiggins2 Did you get time look at the review comments ?

**Comment by latran** [comment]
@ColeHiggins2 any updates on this?

**Comment by jyejare** [comment]
Ping @ColeHiggins2 !!

**Comment by jyejare** [comment]
@lhellebr You closed this but without mentioning the reason. Could you or @ColeHiggins2 please mention the reason as well.

**Pull request closed without merging**